site/src/pages/docs/advanced-production-checklist.mdx

```diff
@@ -96,3 +96,20 @@ See [Using CircuitBreaker with non-Armeria client](/docs/client-circuit-breaker#
   ClientBuilder cb = Clients.builder(...);
   cb.factory(cf);
   ```
+
+- Increase `maxNumEventLoopsPerEndpoint` and `maxNumEventLoopsPerHttp1Endpoint` when your clients send requests
+  to a small number of servers. As an Endpoint uses a single event loop, there may be a bottleneck.
```

Comment:
- What do you think of adding `Consider` since performance may not always be better.
- What do you think of conveying why there is a bottleneck?

```suggestion
- Consider increasing `maxNumEventLoopsPerEndpoint` and `maxNumEventLoopsPerHttp1Endpoint` for better request throughput. Doing so may increase performance at the cost of opening more connections.
```

Reply:
I saw this comment late https://github.com/line/armeria/pull/5297#discussion_r1391916506 . 
I think using this comment is fine, but still I think adding "Consider" is a good idea